src/module.c

```diff
@@ -1530,6 +1530,44 @@ int AlterIndexCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
   return REDISMODULE_OK;
 }
 
+int ConfigCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+  // Not bound to a specific index, so...
+  RedisModule_AutoMemory(ctx);
+
+  // CONFIG <GET|SET> <NAME> [value]
+  if (argc < 3) {
+    return RedisModule_WrongArity(ctx);
+  }
+  const char *action = RedisModule_StringPtrLen(argv[1], NULL);
+  const char *name = RedisModule_StringPtrLen(argv[2], NULL);
+  if (!strcasecmp(action, "GET")) {
+    RSConfig_DumpProto(&RSGlobalConfig, &RSGlobalConfigOptions, name, ctx, 0);
+  } else if (!strcasecmp(action, "HELP")) {
+    RSConfig_DumpProto(&RSGlobalConfig, &RSGlobalConfigOptions, name, ctx, 1);
+  } else if (!strcasecmp(action, "SET")) {
+    size_t offset = 3;  // Might be == argc. SetOption deals with it.
```

Comment:
wouldn't it be easier to deal with 'offset == argc' here? unless there are cases when you do not need value to set configuration but in this case how do you unset it?

Reply:
Yeah, that was the idea. Maybe we'd have an unset, or maybe SET without arguments would restore it to the default.